data-shaping/main.py

```diff
@@ -0,0 +1,26 @@
+from ingest.ingest import read_spreadsheet 
+from output.output import transfer_data, transfer_leave_of_absence_record, transfer_support_record, transfer_trainee_contact, transfer_annual_review_progression, transfer_employment_record, transfer_exit_assessment_record, transfer_trainees, transfer_mid_review_progression
+import os
+
+
+INPUT_DATA_TABLES_PATH = 'data/old_model/SampleData_DataTables.xlsx'
+INPUT_REFERENCE_TABLES_PATH = 'data/old_model/SampleData_ReferenceTables.xlsx'
+
+OUTPUT_DATABASE_PATH = './data/new_model/Database.xlsx'
+
+data_tables = read_spreadsheet(INPUT_DATA_TABLES_PATH)
+reference_tables = read_spreadsheet(INPUT_REFERENCE_TABLES_PATH)
+
+if os.path.exists(OUTPUT_DATABASE_PATH):
```

Comment:
Note: We should either be checking if the directory exists or we should supply this dir' ourselves.

Reply:
The intention behind this was to remove the output xlsx model before every run in order not to have duplication.

In terms of if the folder structure, it is guaranteed since it is a part of the repository. The xlsx files are git ignored as well in order not to put it into Version Control. 

Was this a good understanding of your comment?

Reply:
To be fair, I'm not sure what you're testing this on but the above directorys are needed to be made by hand of `old_model` and `new_model`, the new_model and old_model folders do not exist. 

I'm mentioning that because old_model does not exist, there's a write error.